src/fides/cli/commands/generate.py

```diff
@@ -111,6 +111,52 @@ def generate_dataset_bigquery(
     )
 
 
+@generate_dataset.group(name="aws")
+@click.pass_context
+def generate_dataset_aws(ctx: click.Context) -> None:
```

Comment:
Is this a placeholder for something else?

Reply:
These are the CLI commands to generate a dataset based off of what is discovered in DynamoDB - somewhat opposite to the above comment actually where we were going to make the API grouping be specific to DynamoDB this one might make sense to keep as AWS for now 🤔 could probably use some wider thinking tbh

Reply:
Of course! Now the codebases are unified it makes sense that we'd add this too — nice job!